package.json

```diff
@@ -6,25 +6,20 @@
   },
   "packageManager": "pnpm@7.14.1",
   "scripts": {
-    "lerna": "lerna",
-    "start": "pnpm watch",
-    "dev": "pnpm start",
-    "test": "pnpm --filter \"@trpc/server\" exec jest",
+    "build": "turbo --filter=\"@trpc/*\" build",
+    "build:www": "turbo --filter www build",
```

Comment:
Nit, but we should use `-` instead of `:` --- a bit implicit but we use `:` whenever we wanna run serial or parallel jobs with `npm-run-all`

Reply:
```suggestion
    "build-www": "turbo --filter www build",
```